programs/perpetuals/src/math.rs

```diff
@@ -353,6 +353,19 @@ where
     }
 }
 
+pub fn checked_as_f64<T>(arg: T) -> Result<f64>
```

Comment:
OK clean. I think in another PR we might want to use that for any u64 -> f64 casts throughout the codebase, weird their audit didn't pick it up?